### ISSUE TITLE
"Version in "./docker-compose.yml" is unsupported" on Ubuntu 20.04 LTS

### ISSUE THREAD

**Issue by tvwerkhoven** [LibrePhotos/librephotos#311]
**Error**

> ERROR: Version in "./docker-compose.yml" is unsupported. You might be seeing this error because you're using the wrong Compose file version. Either specify a supported version (e.g "2.2" or "3.3") and place your service definitions under the `services` key, or omit the `version` key and place your service definitions at the root of the file to use version 1.
> For more on the Compose file format versions, see https://docs.docker.com/compose/compose-file/

**Solution**

Change `version` field in [librephoto `docker-compose.yml`](https://raw.githubusercontent.com/LibrePhotos/librephotos-docker/main/docker-compose.yml) from `'3.8'` to `'3.7'`. Looking at [the `docker-compose` changelog](https://docs.docker.com/compose/compose-file/), this version can probably be even lower.

**Comment by zlsl** [comment]
Update your docker-compose to latest
[https://docs.docker.com/compose/install/](https://docs.docker.com/compose/install/)

**Comment by tvwerkhoven** [comment]
> Update your docker-compose to latest
> https://docs.docker.com/compose/install/

Yes that's possible, but that requires installing `docker-compose` outside the package manager. Since there's no requirement to have version 3.8, it's easier to reduce the version requirement.

**Comment by derneuere** [comment]
No longer applicable — the compose files under `deploy/compose/` have no `version:` key at all, and the project targets the `docker compose` v2 CLI rather than the standalone v1 binary this error came from.
